### ISSUE TITLE
Screenlocking on games when application isnt even open, unremovable?

### ISSUE THREAD

**Issue by Shappie112** [blaberry/FullscreenLock#15]
I recently had started using this to use in tandem with borderless gaming while playing Ghostbusters The Video Game Remastered, as it wouldn't let me change cursors from one monitor to another while the game was open. I was able to run it in a window, but then the cursor would drift to other monitors while playing, hence getting Fullscreenlock. It then behaved exactly how I wanted. When the game had focus, the mouse would stay, and I could freely alt-tab out by hitting my Windows key, alt tabbing, etc.

My issue is that now I'm getting pseudo cursor locked in other games. Specifically, I'm having an issue with Warhammer 40k Darktide. I used to simply hit my Windows key to quickly alt tab and then move my cursor to another monitor, but that no longer works. The cursor is locked into the monitor unless I bring up an in-game menu that shows the cursor, then I can just move it off to other monitors freely, which I shouldn't be able to do either. Fullscreenlock isn't open or toggled during this, and nothing changes if it is. 

Basically, Darktide worked exactly how I wanted it to work until I used fullscreenlock for another game to make it work the same way. Now Darktide will not behave properly and it's extremely annoying to do a simple thing like alt tab. All I've done is run this exe for another game and I see no way to uninstall whatever it's done. Please help!

**Comment by Shappie112** [comment]
I have since deleted Fullscreenlock and I'm still getting the weird behaviors from my game. I've reinstalled Darktide and it's still locking my monitor despite fullscreenlock being deleted. I've restarted my computer as well. I've even just done a system restore to a point before I had ever downloaded or run this program and somehow it was still doing this. 

How do I uninstall whatever this application has done? My next idea is to completely reinstall windows, which I'd really rather not have to do.

**Comment by Shappie112** [comment]
Additional odd behavior, when I have fullscreenlock open and I toggle it paused, I still am unable to move my cursor off the screen as I normally should. It acts in the same way as described in issue #13 with it rapidly appearing to move to the other monitor and getting moved back.

Further testing is still only showing this happening with Darktide, which is extremely frustrating for me as I play/stream it a lot and frequently need to switch to other monitors. Loading screens for example I am now locked in to and cannot alt tab to another monitor until they finish loading.

I've also tried using all of the forks and still get the same behavior. There seems to be something that got installed when it was toggled the first time and just isn't going away. Any help would be very appreciated, I'm out of ideas.

**Comment by blaberry** [comment]
The  behaviour you're describing sounds identical to a hack a lot of games use instead of properly locking the mouse to the screen: they just reset the mouse to be within the screen border every tick. This is also the case in CSGO (not sure if they fixed it for CS2), basically once per tick, the mouse is set to the center of the screen, which is why this issue mostly affects high-sens players and awpers in CS. I guess darktide just does the same, but only tries to keep the mouse within the borders of the window instead of trying to reset the mouse to the center.
